src/Components/App/App.js

```diff
@@ -81,7 +81,7 @@ class App extends Component {
 
   //   number 57 In App.js create a method called updatePlaylistName with the following functionality: Accepts a name argument, Sets the state of the playlist name to the input argument
   updatePlaylistName(name) {
-    this.setState({ playlistName: name.target.value });
+    this.setState({ playlistName: name });
```

Comment:
The value of `name` here is going to be whatever you call the `onNameChange` prop with in `Playlist.js`. You _could_ call it with an event (which would have a target and a value) but you it's probably better to get the name off the event at the Playlist level and just call this with the actual name as a string